app/views/ideas/index.html.erb

```diff
@@ -28,6 +29,7 @@
         <td><%= idea.benefits %></td>
         <td><%= idea.impact %></td>
         <td><%= idea.involvement %></td>
+        <td><%= User.find_by(id: idea.assigned_user_id).email unless User.find_by(id: idea.assigned_user_id).blank? %></td>
```

Comment:
We can use - 
`idea.assigned_user&.email `
& will return nil rather than no method error